build.gradle

```diff
@@ -189,6 +189,8 @@ dependencies {
         exclude module: 'fastparse_2.12'
     }
 
+    compile group: 'org.java-websocket', name: 'Java-WebSocket', version: '1.4.0'
```

Comment:
Interesting choice, I justed Tyrus (now part of Eclipse jakarta EE stuff) a few years ago when I started implenenting the sharelatex connection

Reply:
Cool! :) Deciding on using a specific implementation was not so easy since there exist quite a lot implementations, but the used one has a high relevance ranking at mavenrepository.com, it is well maintained and has some nice features.